### ISSUE TITLE
Realtime timeline is faster than before

### ISSUE THREAD

**Issue by Irbidan** [sokcuri/TweetDeckPlayer#89]
I can't read my timeline easily because it is faster. I also notice that some tweets collapse and collide with each other. I remember that was slower reading new tweets using the official app that Twitter offered.

**Comment by sokcuri** [comment]
Is Enable Twitter filter rule function? This function is unstable so i remove this after releases.

**Comment by sokcuri** [comment]
If not, Can i provide more information of issue?

**Comment by Irbidan** [comment]
No, is not checked, i', not using any new options of your TweetDeck. Maybe is my gaming-PC with Windows 10 64 bits updated or i don't know. I'm not following a lot of people on Twitter neither.

**Comment by sokcuri** [comment]
Okay. I will exclude suspicious code from the next update.
